api/src/main/resources/workbench-api.yaml

```diff
@@ -2347,6 +2347,15 @@ paths:
           description: 'The Institution created by this operation.'
           schema:
             "$ref": "#/definitions/Institution"
+        400:
+          description: Bad request
+          schema:
+            "$ref": "#/definitions/ErrorResponse"
+        409:
+          description: Institution with same display Name already exist
```

Comment:
Something like:

DataIntegrityViolationException was thrown, possibly due to an attempt to create a duplicate Institution.